ivadomed/loader/bids_dataframe.py

```diff
@@ -52,6 +52,11 @@ def __init__(self, loader_params: dict, path_output: str, derivatives: bool, spl
         if self.roi_suffix is not None:
             self.target_suffix.append(self.roi_suffix)
 
+        if 'bids_validate' not in loader_params:
+            self.bids_validate = True
+        else:
+            self.bids_validate = loader_params['bids_validate']
+
```

Comment:
I guess we could use `get()` 
```suggestion
            self.bids_validate = loader_params.get('bids_validate', True)
```